docs/02.API-REFERENCE.md

```diff
@@ -797,6 +797,34 @@ typedef void (*jerry_external_string_free_callback_t) (jerry_char_t *string_p,
 - [jerry_create_external_string](#jerry_create_external_string)
 - [jerry_create_external_string_sz](#jerry_create_external_string_sz)
 
+## jerry_external_arraybuffer_free_callback_t
+
+**Summary**
+
+Free callback for external arraybuffers. See
+[jerry_arraybuffer_set_external_free_callback](#jerry_arraybuffer_set_external_free_callback)
+for more information.
+
+**Prototype**
+
+```c
+typedef void (*jerry_external_arraybuffer_free_callback_t) (uint32_t length,
+                                                            void *buffer_p,
+                                                            void *user_p);
+```
+
+- `length` - extrenal buffer length
```

Comment:
What does the "external buffer length" mean? Number of bytes? Number of elements?